cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       labels:
         k8s-app: heapster
         version: v1.1.0
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
Setting it to "true" implies that "false" means the opposite, but it doesn't (AFAIK).  We could instead demand that it == "true", which is what we have done for other annotations, but then you need to log errors/events if it == "True" or "1" or "yes" or "purple".

I don't feel that strongly since it is alpha and we are inconsistent elsewhere already.